vtex-apps/listrak-pixel
# [LISTRAK-11] Add option to send SKU ref ID instead of SKU ID to Listrak

**What problem is this solving?**
Client wanted option to send Listrak the SKU reference ID. Added toggle in app settings to send SKU reference ID when enabled and VTEX SKU ID when disabled.

<img width="525" alt="Screen Shot 2022-06-27 at 4 33 45 PM" src="https://user-images.githubusercontent.com/53097865/176030771-923a58a4-2ab1-4e33-8d0d-d467bf6885c7.png">

**How to test it?**
Test in account [sandboxusdev](https://sandboxusdev.myvtex.com/admin) (already installed a beta version of the app with this functionality)

1. In admin, update a SKU (e.g. shipment weight) and let it reindex
2. Check Splunk for `ProductHandler-createProductResponse` log to see SKU info sent to Listrak

_Example_
<img width="997" alt="Screen Shot 2022-06-27 at 4 28 47 PM" src="https://user-images.githubusercontent.com/53097865/176030050-f725941d-702c-42f9-aadf-148ebbdde724.png">
